cmd/status.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/codegangsta/cli"
+	"github.com/exercism/cli/api"
+	"github.com/exercism/cli/config"
+)
+
+// Status command allows a user to view their progress in a given language track
+func Status(ctx *cli.Context) {
+	c, err := config.New(ctx.GlobalString("config"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	args := ctx.Args()
+
+	if len(args) != 1 {
+		msg := "Usage: exercism status LANGUAGE"
+		log.Fatal(msg)
+	}
+
+	language := args[0]
+	client := api.NewClient(c)
+	status, err := client.Status(language)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	printResponse(language, status)
+}
+
+func printResponse(language string, status [][]string) {
+	recent := status[0]
+	fetched := status[1]
+	skipped := status[2]
+	fmt.Println("\nYour status on the", language, "track:")
+
+	fmt.Println("\nMost recently submitted exercise")
+	fmt.Println(recent[0], "( submitted on", recent[1], ")")
+
+	fmt.Println("\nExercises fetched but not submitted")
+	for _, exercise := range fetched {
+		fmt.Println(exercise)
+	}
+	fmt.Println("\nExercises skipped")
+	for _, exercise := range skipped {
+		fmt.Println(exercise)
+	}
+	fmt.Println("")
```

Comment:
If we have a custom status type, then we can move the printing/formatting into a String() function and do a `fmt.Println(status)`.